cloud/amazon/ec2_asg.py

```diff
@@ -226,428 +218,557 @@
     region: us-east-1
 '''
 
-import time
 import logging as log
-import traceback
 
 from ansible.module_utils.basic import *
 from ansible.module_utils.ec2 import *
-log.getLogger('boto').setLevel(log.CRITICAL)
-#log.basicConfig(filename='/tmp/ansible_ec2_asg.log',level=log.DEBUG, format='%(asctime)s: %(message)s', datefmt='%m/%d/%Y %I:%M:%S %p')
 
+log.getLogger('boto').setLevel(log.CRITICAL)
+# log.basicConfig(filename='/tmp/ansible_ec2_asg.log', level=log.DEBUG, format='%(asctime)s: %(message)s',
+#                 datefmt='%m/%d/%Y %I:%M:%S %p')
 
 try:
-    import boto.ec2.autoscale
-    from boto.ec2.autoscale import AutoScaleConnection, AutoScalingGroup, Tag
-    from boto.exception import BotoServerError
-    HAS_BOTO = True
+    import boto3
+    import botocore
+
+    HAS_BOTO_3 = True
 except ImportError:
-    HAS_BOTO = False
+    HAS_BOTO_3 = False
 
-ASG_ATTRIBUTES = ('availability_zones', 'default_cooldown', 'desired_capacity',
-    'health_check_period', 'health_check_type', 'launch_config_name',
-    'load_balancers', 'max_size', 'min_size', 'name', 'placement_group',
-    'termination_policies', 'vpc_zone_identifier')
+ASG_ATTRIBUTES = ('AvailabilityZones', 'DefaultCooldown', 'DesiredCapacity',
+                  'HealthCheckGracePeriod', 'HealthCheckType', 'LaunchConfigurationName',
+                  'LoadBalancerNames', 'MaxSize', 'MinSize', 'AutoScalingGroupName', 'PlacementGroup',
+                  'TerminationPolicies', 'VPCZoneIdentifier')
 
 INSTANCE_ATTRIBUTES = ('instance_id', 'health_status', 'lifecycle_state', 'launch_config_name')
 
+
 def enforce_required_arguments(module):
-    ''' As many arguments are not required for autoscale group deletion
+    """ As many arguments are not required for autoscale group deletion
         they cannot be mandatory arguments for the module, so we enforce
-        them here '''
+        them here """
     missing_args = []
     for arg in ('min_size', 'max_size', 'launch_config_name'):
         if module.params[arg] is None:
             missing_args.append(arg)
     if missing_args:
-        module.fail_json(msg="Missing required arguments for autoscaling group create/update: %s" % ",".join(missing_args))
+        module.fail_json(
```

Comment:
This change seems to be style-only, can you revert it since you aren't changing any actual code on this line?